docs/langref/src/SUMMARY.md

```diff
@@ -43,6 +43,10 @@
     -   [`Math` Namespace](math.md)
     -   [`Colors` Namespace](colors.md)
 -   [Font Handling](fonts.md)
+-   [Debugging Techniques](debugging_techniques.md)
+    -   [Slow Motion Animations](debug_slomo.md)
+    -   [User Interface Scaling](debug_scaling.md)
+    -   [Performance Debugging](debug_performance.md)
```

Comment:
I don’t think that these make sense in this book. The advice is useless unless you’re debugging a native application. This isn’t actionable when using the live preview for example.